src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -16,15 +16,10 @@
 
 package com.google.codeu.data;
 
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.*;
```

Comment:
I think it is best practice to import specifically what you need instead of using import .*

Reply:
That's right.